### PR TITLE
WIP #6 mute upsert

### PR DESCRIPTION
**Pull request by davidv171**
RFC if this is the best way to do this. If we want to keep all history of mutes, this might be the way to go.

### CODE REVIEW

File: src/db/mute.rs [hunk 1]
```diff
@@ -95,7 +96,19 @@ impl Db {
...
+        sqlx::query!(
+            "update mute set active=false where usr=? and active=true",
+            id
+        )
+        .execute(&mut conn)
+        .await?;
+        Ok(())
+    }
+
+    // This is rather unperformant, i'd like to have a cleaner solution that doesn't do a extra request per mute
```
Comment (elkowar) [inline src/db/mute.rs:111]: now that we're not using it for that, we can remove that comment :D
```suggestion
```

**Comment by davidv171** [comment]
Build failing, because I need to run the sqlx command again, will fix when I get home, after that can it get merged?

**Comment by elkowar** [comment]
Thanks!

**Comment by 6gk** [comment]
what a cruel april fools joke.

**Pull request merged**